bin/postman-test/collections/system-management.postman_collection.json

```diff
@@ -209,26 +199,13 @@
               "});",
               "",
               "pm.test(\"Response time is less than 600ms\", function () {",
-              "    pm.expect(pm.response.responseTime).to.be.below(600);",
+              "    pm.expect(pm.response.responseTime).to.be.below(6000);",
```

Comment:
Should this still be 6 seconds (or should it be reverted to 600ms)?

Reply:
- I investigated and it turns out that we still _do_ require a six seconds timeout 👍